Google.PowerShell/ReleaseFiles/PatchFromGcs.ps1

```diff
@@ -0,0 +1,43 @@
+﻿# Copyright 2016 Google Inc. All Rights Reserved.
+# Licensed under the Apache License Version 2.0.
+#
+# Updates Cloud Tools for PowerShell module to the latest found in Google Cloud Storage bucket g-cloudsharp-unsignedbinaries
+
+param($installPath) # Let a user manually select a Cloud SDK install path
+$installPath = $installPath -or $args[0]
+
+# Find the Google Cloud SDK install path from the registry.
+if (-not $installPath) {
+    $hklmPath = "HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Google Cloud SDK"
+    $hkcuPath = "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Google Cloud SDK"
+    if (Test-Path $hklmPath) {
+        $installPath = Get-ItemPropertyValue $hklmPath InstallLocation
+    } elseif (Test-Path $hkcuPath) {
+        $installPath = Get-ItemPropertyValue $hkcuPath InstallLocation
+    } else {
+        Write-Error "Can not find Cloud SDK from the registry."
+        return
+    }
+}
+$installPath = $installPath -replace '"' # Registry values had quotes for some reason
```

Comment:
"had quotes for some reason" implies you don't know what is going on. (Which may be true.) But to err on the sound of sounding professional, let us just say "The Cloud SDK path is quoted." and leave it at that.